pkg/collect/host_run.go

```diff
@@ -32,16 +38,57 @@ func (c *CollectHostRun) IsExcluded() (bool, error) {
 
 func (c *CollectHostRun) Collect(progressChan chan<- interface{}) (map[string][]byte, error) {
 	runHostCollector := c.hostCollector
+	collectorName := runHostCollector.CollectorName
+	if collectorName == "" {
+		collectorName = "run-host"
+	}
+	var (
+		bundleOutputPath          string
+		bundleOutputRelativePath  string
+		cmdConfigFilePath         string
+		cmdConfigFileRelativePath string
+	)
 
 	cmd := exec.Command(runHostCollector.Command, runHostCollector.Args...)
 
+	cmd.Env = append(os.Environ(),
+		fmt.Sprintf("BUNDLE_PATH=%s", c.BundlePath),
```

Comment:
I'm not sure we want to expose this bundle path. We would not want misbehaving tools to tamper with other collected files.

Reply:
gonna drop this, i forgot why i even added it here 😓